meetups/praha.yml

```diff
@@ -196,3 +196,23 @@
     longitude: '14.4574506'
     url: https://www.google.cz/maps/place/Hotel+Ol%C5%A1anka/@50.0828549,14.4552392,17z/data=!3m1!4b1!4m5!3m4!1s0x470b9361dfdd06b7:0x967cf4b46e46fa30!8m2!3d50.0828549!4d14.4574279
 
+- name: Pyladies Praha <br> Začátečnický kurz úterý
+  materials: http://naucse.python.cz/2018/pyladies-praha-podzim-cznic/
+  parallel-with-previous: true
+  start: 2018-XX-XX
```

Comment:
To `XX` nefunguje. Místo toho celý řádek zakomentuj  – `  #start: 2018-XX-XX`. Měsíc začátku můžeš kdyžtak dát do `time` (to je textové políčko).